### ISSUE TITLE
Permissionless operators should allow "Partial"

### ISSUE THREAD

**Issue by Valdorff** [StakingDirectory/StakingDirectory#38]
The one that comes to mind is Stader. While they do have permissionless operators, they also have permissioned operators that don't need to provide capital (a mix of the RP and Lido models in that sense). This could be visualized with a yellow ~ or something along those lines.

**Comment by EridianAlpha** [comment]
This was also discussed during development. The thought was that if an operator can be permissionless, that information is what is useful to people viewing Staking Directory to decide who to stake with. Any extra flags we add to show information, such as additional colors and symbols will make it much more complex for users to understand and follow.

This information could be more clearly displayed in the expanded sections though, so that is something I will work on designs for. Thanks for your feedback 🙂

**Comment by Valdorff** [comment]
Hmmm... I can understand that take for the (Node) entry. For the (LST) entry, I don't think I agree that a user is looking for whether any are allowed. On the LST front, it's likely related to their ethos and beliefs about decentralization/trustlessness aiding security.

I do understand not wanting to complicate it. My worry is that this isn't "not providing info", but rather "actively providing the wrong info".

One path is having it only in the expanded section where the context doesn't add cognitive load to the summary. I think I like the original thought best still - and I think people are pretrained enough on green/yellow/red - but it's ultimately not up to me 😛

**Comment by EridianAlpha** [comment]
Yes I see, the (Node) entry was the focus as someone mainly needs to know if it's permissionless or not for them to run a node with that provider. But for (LST) entries, a more helpful metric would be how much of the total stake is run by permissionless operators (if any). That information might get difficult to compare though, so for this case, a middle category of "permissioned and permissionless" operators would make sense.

A simple way to achieve this would be an orange tick on the checklist, with a different explanation. Currently, the checklist is coded with Boolean true/false values so it's not a quick change to add, but something I will work on and update you about here.

Thanks again for your input as this is a community project, so your feedback is greatly appreciated!
